src/node/config.ts

```diff
@@ -396,6 +399,10 @@ export async function resolveConfig(
       // transpile es import syntax to require syntax using rollup.
       const rollup = require('rollup') as typeof Rollup
       const esbuildPlugin = await createEsbuildPlugin({})
+      const esbuildRenderChunkPlugin = createEsbuildRenderChunkPlugin(
+        'es2019',
```

Comment:
Instead of hardcoded, should we read it from tsconfig.json and fallback to something like `es2019`?